content/intro-to-storybook/react/it/screen.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: 'Construct a screen'
-tocTitle: 'Screens'
-description: 'Construct a screen out of components'
+title: 'Costruisci una schermata'
+tocTitle: 'Schermate'
+description: 'Costruisci una schermata da componenti'
 commit: 'bb2471f'
 ---
 
-We've concentrated on building UIs from the bottom up, starting small and adding complexity. Doing so has allowed us to develop each component in isolation, figure out its data needs, and play with it in Storybook. All without needing to stand up a server or build out screens!
+Ci siamo concentrati sulla costruzione di interfacce utente dal basso verso l'alto, partendo da piccole e aggiungendo complessità. Farlo ci ha permesso di sviluppare ogni componente in isolamento, capire le sue esigenze di dati e giocare con esso in Storybook. Tutto senza dover avviare un server o costruire schermate!
 
-In this chapter, we continue to increase the sophistication by combining components in a screen and developing that screen in Storybook.
+In questo capitolo, continuiamo ad aumentare la sofisticazione combinando i componenti in una schermata e sviluppando quella schermata in Storybook.
```

Comment:
I think **sofisticazione** it's better if changed with **sofisticatezza** in this case.